### PR TITLE
[PR] Adds PWA customization

### LINKED ISSUE #80: Research: How to create a Progressive Web App (PWA) using `Flutter`
At present I am using the [**`MVP`**](https://mvp.fly.dev/) in `Safari` Desktop and Mobile on my Mac and iPhone respectively.

image

It's an "OK" experience during `MVP` ... but we can do _much_ better even in the _short_-term. 

As we recently saw with **`Hoppscotch`**: [/dwyl/mvp/hoppscotch.md#hoppscotch-setup](https://github.com/dwyl/mvp/blob/eafe737642cced060eac9862c0b7b2c471d2909a/lib/api/hoppscotch.md#hoppscotch-setup)
The recommended approach to running their App is to "Install" it as a "Chrome" App:

image

Which is exactly how I run it on my `localhost`:

image

It means that it runs full-screen and you can <kbd>command | alt</kbd>+<kbd>tab</tab> and immediately find it:

image

6 Second Demo:
https://user-images.githubusercontent.com/194400/219318400-e5a14779-ccb3-479f-a8fc-73700323016a.mov

# Todo

+ [x] Research how to package a `Flutter App` as a `PWA` 🔍 e.g: https://blog.codemagic.io/pwa-in-flutter/
+ [ ] Create new file: `pwa.md` in _this_ repo and capture all the knowledge + steps required. 📝 
+ [ ] Add `PWA` capability to one of our _existing_ `Flutter` demo apps 📱 e.g: https://github.com/dwyl/flutter-phoenix-channels-demo

### PR DESCRIPTION
**Pull request by LuchoTurtle**
closes https://github.com/dwyl/learn-flutter/issues/80

This adds a document `pwa.md` in the `app` folder detailing how to change the icon, text and overall enhance the `PWA` experience.

### CODE REVIEW

**Comment by LuchoTurtle** [comment]
This PR should be reviewed in conjunction with https://github.com/dwyl/learn-flutter/pull/81, as they're both related.

It's ready to be merged, having reviewed the changes and the README.

File: README.md [hunk 1]
```diff
@@ -1000,6 +1000,32 @@ You can find more information
...
+
+# Enhancing the `PWA`
+
+Since we've used `Flutter`,
+we can leverage its capabilities to deploy it
+to iOS, Android and Web devices.
+
+For the latter, 
+when creating the **release bundle**,
+the user is *able to download a [`PWA`](https://web.dev/progressive-web-apps/)*.
```
Comment (nelsonic) [inline README.md]:
```suggestion
the person is able to download a [`PWA`](https://web.dev/progressive-web-apps/) 
(Progressive Web App).
```

File: app/pwa.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+
+```sh
+flutter pub run flutter_launcher_icons
+```
+
+The terminal will output 
+this information:
+
+```sh
+ase) lucho@Luiss-MBP app % flutter pub run flutter_launcher_icons
```
Comment (nelsonic) [inline app/pwa.md]:
```suggestion
flutter pub run flutter_launcher_icons
```

**Review by nelsonic — APPROVED** [review]
Thanks @LuchoTurtle 🥇

**Pull request merged**